src/olympia/abuse/utils.py

```diff
@@ -60,10 +72,37 @@ def send_mail(self, subject, message, recipients):
         send_mail(subject, message, recipient_list=[user.email for user in recipients])
 
     def notify_reporters(self):
+        template = loader.get_template(self.reporter_template_path)
         for abuse_report in self.cinder_job.abusereport_set.all():
-            if abuse_report.reporter or abuse_report.reporter_email:
-                # TODO: notify reporter
-                pass
+            email_address = (
+                abuse_report.reporter.email
+                if abuse_report.reporter
+                else abuse_report.reporter_email
+            )
+            if not email_address:
+                continue
+            with translation.override(
+                abuse_report.application_locale or settings.LANGUAGE_CODE
```

Comment:
I'll file a follow-up so the other abuse report serializers also store `lang` as `application_locale` (currently only AddonAbuseSerializer does)

Reply:
https://github.com/mozilla/addons-server/issues/21529